src/Avalonia.Animation/Animatable.cs

```diff
@@ -14,26 +14,14 @@ namespace Avalonia.Animation
     /// Base class for all animatable objects.
     /// </summary>
     public class Animatable : AvaloniaObject
-    { 
-        /// <summary>
-        /// Defines the <see cref="PlayState"/> property.
-        /// </summary>
-        public static readonly DirectProperty<Animatable, PlayState> PlayStateProperty =
-            AvaloniaProperty.RegisterDirect<Animatable, PlayState>(
-                nameof(PlayState),
-                o => o.PlayState,
-                (o, v) => o.PlayState = v);
-
-        private PlayState _playState = PlayState.Run;
+    {
+        public static readonly StyledProperty<IClock> ClockProperty =
+            AvaloniaProperty.Register<Animatable, IClock>(nameof(Clock), inherits: true);
 
-        /// <summary>
-        /// Gets or sets the state of the animation for this
-        /// control.
-        /// </summary>
-        public PlayState PlayState
+        public IClock Clock
```

Comment:
So if I'm understanding this correctly, an animation clock an be applied to a control and that clock can be used to control the animation state for the sub-tree? That's a pretty cool idea! Could we get some XML docs explaining what this does?

Reply:
One question: what would happen if you change the clock while animations are active? Nothing I assume?

Reply:
Yeah if you change the clock while an animation is running then it won't change it. Also, because of the way `TransformAnimator` works, it won't actually let you change clocks after `Apply` is called. I can change it so that this limitation doesn't happen, but the API becomes a little less clean.

Reply:
Yeah it's fine as it is, I think. Just wanted to clarify. Lets leave it for now and revisit if it causes confusion.